mne/io/edf/tests/test_edf.py

```diff
@@ -97,6 +97,13 @@ def test_bdf_data():
     assert (raw_py.info['chs'][63]['loc']).any()
 
 
+@testing.requires_testing_data
+def test_bdf_crop_save_stim_channel():
+    """Test EDF with various sampling rates."""
+    raw = read_raw_bdf(bdf_stim_channel_path)
+    raw.save('./test-raw.fif', tmin=1.2, tmax=4.0, overwrite=True)
```

Comment:
Just one minor thing, we usually don't use './' so you could remove it. Also, `tmin` and `tmax` are not needed (the problem also occurred without them).